openshift/e2e-tests-openshift.sh

```diff
@@ -0,0 +1,215 @@
+  local tests_finished=0
+  for i in {1..60}; do
+    local finished="$(kubectl get $1.tekton.dev --output=jsonpath='{.items[*].status.conditions[*].status}')"
+    if [[ ! "$finished" == *"Unknown"* ]]; then
+      tests_finished=1
+      break
+    fi
+    sleep 10
+  done
+
+  return ${tests_finished}
+}
+
+function check_results() {
+  local failed=0
+  results="$(kubectl get $1.tekton.dev --output=jsonpath='{range .items[*]}{.metadata.name}={.status.conditions[*].type}{.status.conditions[*].status}{" "}{end}')"
+  for result in ${results}; do
+    if [[ ! "${result,,}" == *"=succeededtrue" ]]; then
+      echo "ERROR: test ${result} but should be succeededtrue"
+      failed=1
+    fi
+  done
+
+  return ${failed}
+}
+
+function output_yaml_test_results() {
+  # If formatting fails for any reason, use yaml as a fall back.
+  oc get $1.tekton.dev -o=custom-columns-file=${REPO_ROOT_DIR}/test/columns.txt || \
+    oc get $1.tekton.dev -oyaml
+}
+
+function output_pods_logs() {
+    echo ">>> $1"
+    oc get $1.tekton.dev -o yaml
+    local runs=$(kubectl get $1.tekton.dev --output=jsonpath="{.items[*].metadata.name}")
+    set +e
+    for run in ${runs}; do
+	echo ">>>> $1 ${run}"
+	case "$1" in
+	    "taskrun")
+		go run ./test/logs/main.go -tr ${run}
+		;;
+	    "pipelinerun")
+		go run ./test/logs/main.go -pr ${run}
+		;;
+	esac
+    done
+    set -e
+    echo ">>>> Pods"
+    kubectl get pods -o yaml
+}
+
+function delete_build_pipeline_openshift() {
+  echo ">> Bringing down Build"
+  oc delete --ignore-not-found=true -f tekton-pipeline-resolved.yaml
+  # Make sure that are no residual object in the tekton-pipelines namespace.
+  oc delete --ignore-not-found=true taskrun.tekton.dev --all -n $TEKTON_PIPELINE_NAMESPACE
+  oc delete --ignore-not-found=true pipelinerun.tekton.dev --all -n $TEKTON_PIPELINE_NAMESPACE
+  oc delete --ignore-not-found=true task.tekton.dev --all -n $TEKTON_PIPELINE_NAMESPACE
+  oc delete --ignore-not-found=true clustertask.tekton.dev --all -n $TEKTON_PIPELINE_NAMESPACE
+  oc delete --ignore-not-found=true pipeline.tekton.dev --all -n $TEKTON_PIPELINE_NAMESPACE
+  oc delete --ignore-not-found=true pipelineresources.tekton.dev --all -n $TEKTON_PIPELINE_NAMESPACE
+}
+
+function delete_test_resources_openshift() {
+  echo ">> Removing test resources (test/)"
+  oc delete --ignore-not-found=true -f tests-resolved.yaml
+}
+
+ function delete_test_namespace(){
+   echo ">> Deleting test namespace $TEST_NAMESPACE"
+   #oc policy remove-role-from-group system:image-puller system:serviceaccounts:$TEST_NAMESPACE -n $OPENSHIFT_BUILD_NAMESPACE
+   #oc delete project $TEST_NAMESPACE
+   oc policy remove-role-from-group system:image-puller system:serviceaccounts:$TEST_YAML_NAMESPACE -n $OPENSHIFT_BUILD_NAMESPACE
+   oc delete project $TEST_YAML_NAMESPACE
+ }
+
+function teardown() {
+  delete_test_namespace
+  delete_test_resources_openshift
+  delete_build_pipeline_openshift
+}
+
+create_test_namespace
+
+enable_docker_schema2
```

Comment:
Do we still need this?

Reply:
Looking in `knative-serving`, we don't :angel: